crates/re_renderer/shader/colormap.wgsl

```diff
@@ -0,0 +1,115 @@
+#import <./types.wgsl>
+#import <./utils/srgb.wgsl>
+
+// NOTE: Keep in sync with `colormap.rs`!
+const GRAYSCALE:        u32 = 0u;
+const COLORMAP_TURBO:   u32 = 1u;
+const COLORMAP_VIRIDIS: u32 = 2u;
+const COLORMAP_PLASMA:  u32 = 3u;
+const COLORMAP_MAGMA:   u32 = 4u;
+const COLORMAP_INFERNO: u32 = 5u;
+
+fn colormap_srgb(which: u32, t: f32) -> Vec3 {
```

Comment:
should document that to is expected to be 0 to 1. (does anything meaningful happen if it isn't?)

Reply:
Should also document the output range (0-1? 0-255?)

Reply:
> should document that to is expected to be 0 to 1. (does anything meaningful happen if it isn't?)

I think I'm going to add a `saturate()` everywhere for the GPU ones